src/Components/CreateImageWizardV2/utilities/requestMapper.tsx

```diff
@@ -0,0 +1,150 @@
+import { store } from '../../../store';
+import {
+  AwsUploadRequestOptions,
+  CreateBlueprintRequest,
+  Customizations,
+  GcpUploadRequestOptions,
+  ImageRequest,
+  ImageTypes,
+  Subscription,
+  UploadTypes,
+} from '../../../store/imageBuilderApi';
+import { wizardState } from '../../../store/wizardSlice';
+
+/**
+ * This function maps the wizard state to a valid CreateBlueprint request object
+ * @param {string} orgID organization ID
+ * @returns {CreateBlueprintRequest} creation blueprint request payload
+ */
+export const mapRequestFromState = (orgID: string): CreateBlueprintRequest => {
+  const { wizard: wizardState } = store.getState();
+  const images = getImageRequest(wizardState);
+  const customizations = getCustomizations(wizardState, orgID);
+
+  return {
+    name: wizardState.details.blueprintName,
```

Comment:
For the name, the description, and distribution can we use a selector function instead? We will just need to define custom selectors in the wizardSlice for this. That way the shape of the wizardState is decoupled from the rest of the application. So something like:
```javascript
export const selectBlueprintName = (state: RootState) => {
  return state.details.blueprintName;
};
```